<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="loadItemsAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="drawingWallAction"/>
    <addaction name="drawingWindowAction"/>
    <addaction name="drawingDoorAction"/>
    <addaction name="drawingChairAction"/>
    <addaction name="drawingBedAction"/>
    <addaction name="drawingPianoAction"/>
    <addaction name="drawingDeskAction"/>
    <addaction name="separator"/>
    <addaction name="eraseAction"/>
    <addaction name="clearAction"/>
    <addaction name="drawingWardrobeAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Интерфейс</string>
    </property>
    <addaction name="editPropertiesAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="saveAction"/>
   <addaction name="saveAsAction"/>
   <addaction name="loadItemsAction"/>
   <addaction name="separator"/>
   <addaction name="eraseAction"/>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="drawingWallAction"/>
   <addaction name="drawingWindowAction"/>
   <addaction name="drawingDoorAction"/>
   <addaction name="separator"/>
   <addaction name="drawingChairAction"/>
   <addaction name="drawingBedAction"/>
   <addaction name="drawingPianoAction"/>
   <addaction name="drawingDeskAction"/>
   <addaction name="drawingWardrobeAction"/>
  </widget>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/toXml.png</normaloff>
     <activeon>:/icons/resourses/icons/toXml_enabled.png</activeon>:/icons/toXml.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить схему</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить схему как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="loadItemsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/fromXml.png</normaloff>
     <activeon>:/icons/resourses/icons/fromXml_enabled.png</activeon>:/icons/fromXml.png</iconset>
   </property>
   <property name="text">
    <string>Загрузить схему</string>
   </property>
   <property name="toolTip">
    <string>Загрузить схему из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="editPropertiesAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/properties.png</normaloff>
     <activeon>:/icons/resourses/icons/settings_enabled.png</activeon>:/icons/properties.png</iconset>
   </property>
   <property name="text">
    <string>Показать панель свойств элементов</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="drawingWallAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/wall.png</normaloff>:/icons/wall.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать стену</string>
   </property>
  </action>
  <action name="eraseAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/erase_action.png</normaloff>:/icons/erase_action.png</iconset>
   </property>
   <property name="text">
    <string>Удалить графический элемент схемы</string>
   </property>
  </action>
  <action name="drawingWindowAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/window.png</normaloff>:/icons/window.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать окно</string>
   </property>
  </action>
  <action name="drawingDoorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/door.png</normaloff>:/icons/door.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать дверной проем</string>
   </property>
  </action>
  <action name="drawingBedAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/bed.png</normaloff>:/icons/bed.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать кровать</string>
   </property>
  </action>
  <action name="drawingChairAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/chair.png</normaloff>:/icons/chair.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать стул</string>
   </property>
  </action>
  <action name="drawingDeskAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/desk.png</normaloff>:/icons/desk.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать письменный стол</string>
   </property>
  </action>
  <action name="drawingPianoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/piano.png</normaloff>:/icons/piano.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать пианино</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить холст</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
  </action>
  <action name="drawingWardrobeAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/wardrobe.png</normaloff>:/icons/wardrobe.png</iconset>
   </property>
   <property name="text">
    <string>Нарисовать платяной шкаф</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
